core/trino-main/src/test/java/io/trino/server/ui/TestWebUi.java

```diff
@@ -836,7 +842,31 @@ public void testCustomPrincipalField()
         }
     }
 
-    private void assertAuth2Authentication(TestingTrinoServer server, String accessToken, boolean refreshTokensEnabled)
+    @Test
+    public void testOAuth2AuthenticatorWithoutEndsessionEndpoint()
```

Comment:
What if `http-server.authentication.oauth2.scopes` does not contain `openid` scope but IdP returned `end_session_endpoint` and the other way around?